### PR TITLE
adding howto docs

### PR DESCRIPTION
**Pull request by jessegoodier**
## What does this PR change?
* add a howto for installing on a clean cluster
* fixes some kubecost>opencost naming in files 
* sample queries

## Does this PR relate to any other PRs?
* No 

## How will this PR impact users?
* Easier to install opencost and find sample queries.

## Does this PR address any GitHub or Zendesk issues?
* There are many questions about the APIs in slack, can reply to them with the sample-queries.md

## How was this PR tested?
* gke clusters

## Does this PR require changes to documentation?
* no

## Have you labeled this PR and its corresponding Issue as "next release" if it should be part of the next Kubecost release? If not, why not?
* NA

### CODE REVIEW

File: kubernetes/deployment.yaml [hunk 1]
```diff
@@ -30,7 +30,7 @@ spec:
               memory: "55M"
           env:
             - name: PROMETHEUS_SERVER_ENDPOINT
-              value: "{{prometheusEndpoint}}"  #The endpoint should have the form http://<service-name>.<namespace-name>.svc
+              value: "http://my-prometheus-server.prom.svc"  #prometheusEndpoint should have the form http://<service-name>.<namespace-name>.svc
```
Comment (jessegoodier) [inline kubernetes/deployment.yaml]: I made this change so the install will work if following instructions.

File: deploying-as-a-pod.md [hunk 2]
```diff
@@ -1,8 +1,6 @@
-## Deploying as a pod
+# Deploying as a pod
 
-See this page for all [Kubecost install options](http://docs.kubecost.com/install).
```
Comment (dwbrown2) [inline deploying-as-a-pod.md:3]: How do you feel about keeping a pointer to this new page? https://www.opencost.io/docs/install

This allows us to always keep a single source of truth in terms of full set of install options.

File: docs/basic-setup.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
+# OpenCost Install
```
Comment (dwbrown2) [inline docs/basic-setup.md]: @jessegoodier what are thoughts on moving these and other files in this directory to https://www.opencost.io/docs/?

**Review by dwbrown2 — COMMENTED** [review]
This looks amazing, @jessegoodier! Dropped two quick questions. Can take another pass, just let me know.

File: deploying-as-a-pod.md (hunk 2 above)
Comment (jessegoodier) [reply deploying-as-a-pod.md:3]: For sure, this doc needs a little work. Will add this to the top.

File: docs/basic-setup.md (hunk 3 above)
Comment (jessegoodier) [reply docs/basic-setup.md]: For sure, we need this there. Ideally that would source from this repo though?
I wonder if that is feasible?

If not, maybe this dir can link to that.

**Review by kc-adawson — APPROVED** [review]
Big improvement from previous version

**Comment by jessegoodier** [comment]
I just cleaned up the API samples after feedback from @michaelmdresser

File: docs/basic-setup.md (hunk 3 above)
Comment (dwbrown2) [reply docs/basic-setup.md]: It's actually sourced from this repo currently: https://github.com/kubecost/opencost-website/tree/main/docs

That seem ok?

**Comment by brstuder** [comment]
I would definitely move towards getting this rolled out ASAP, there is a lot of content being added that I may still need to go in and make updates, but cleaning up the installation instructions and moving away from a reliance on Kubecost is a great start.

**Comment by dwbrown2** [comment]
+1, tons of great stuff here! @jessegoodier anything we can help with?

**Comment by brstuder** [comment]
I would be in favor of getting this merged ASAP, should we wait until Jesse gets back next week?

**Comment by dwbrown2** [comment]
That'd be my vote, unless there's a major critical blocker/issue.

**Comment by jessegoodier** [comment]
I cleaned up a few more things and couldn't push to my old branch due to permission changes.

Not sure what the right approach is, but here's a new PR that is from a fork I just created.

New PR: https://github.com/opencost/opencost/pull/1369/files

**Comment by dwbrown2** [comment]
@mbolt35 would you have time to help merge while on support? maintainer approval may be needed now.

**Comment by jessegoodier** [comment]
> @mbolt35 would you have time to help merge while on support? maintainer approval may be needed now.

I'm thinking that maybe I should have merged from my personal account to the branch on this PR? 

Please let me know whichever method is easiest.

**Review by mbolt35 — APPROVED** [review]
These updates look reasonable (I especially like the reference to a single source of documentation versus attempting to replicate docs in so many places). Also, 👍 for the postman collection. Very nice.

**Comment by srpomeroy** [comment]
Do we not have an action to stand up an example site via Vercel? Or am I missing it?

**Pull request closed without merging**